docker/0L/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04
+
+# Todo (nourspace): Add user, create home dir, chmod, etc
+# Todo (nourspace): Pass $OL_ROOT_DIR as env to be used in (ol, tower, ...) commands as --config $OL_ROOT_DIR/0L.toml
+ENV PATH="${PATH}:~/aliases" \
+    OL_ROOT_DIR="~/.0L" \
+    OL_CONFIG="${OL_ROOT_DIR}/.0l.toml"
+
+# Install system prerequisites
+RUN apt-get update && apt-get install -y \
+  curl \
+  && rm -rf /var/lib/apt/lists/*
+
+# Install 0L built binaries [Easy Mode]
+# https://github.com/OLSF/libra/blob/main/ol/documentation/node-ops/validators/validator_onboarding_easy_mode.md
+RUN curl -sL https://raw.githubusercontent.com/OLSF/libra/main/ol/util/install.sh | bash
```

Comment:
Why use an external url (with the risk of injections)? Also, might lead to inconsistent CI/CD. E.g.: you modify install.sh and the pipeline won't pick it up here since it's still reading from main.

Reply:
I had a similar thought. Ideally the dockerfile would reside in the repo root and would build from the same checkout. I /think/ the current notion is that you could run this Dockerfile standalone and get a build of current master @HEAD

Reply:
Hey @emanuelconunaemme this seems to be an older version where I was trying to use the easy mode still. Please check the latest changes. I'm now using git to clone my own fork. But I'm preparing to start cloning upstream using release tag or simply build directly from it.
I'm yet to conclude where this setup should end up being. The main repo or some other tooling repo that we create.